tron/core/actionrun.py

```diff
@@ -165,6 +167,12 @@ def __init__(
         self.output_path = output_path or filehandler.OutputPath()
         self.output_path.append(self.id)
         self.context = command_context.build_context(self, parent_context)
+        self.retries = retries
+        self.retry_codes = retry_codes
```

Comment:
This `retry_codes` kinda confuses me. Can it just be called `exit_statuses` to mirror `exist_status`? And then just always be a list? (never None)

Reply:
i didn't like how plural of status sounds, but you are right. will fix.